common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -223,6 +223,8 @@ public class BrokerConfig extends BrokerIdentity {
     private boolean enablePopBatchAck = false;
     private boolean enableNotifyAfterPopOrderLockRelease = true;
     private boolean initPopOffsetByCheckMsgInMem = true;
+    // read message from pop retry topic v1, for the compatibility, will be removed in the next version
+    private boolean retrieveMessageFromPopRetryTopicV1 = true;
```

Comment:
If retrieveMessageFromPopRetryTopicV1 is later removed, some ecosystem-related proxies (such as MQTT proxy) will need to be synchronized and modified accordingly.